packages/perseus/src/util/answer-types.ts

```diff
@@ -69,7 +69,6 @@ export type Score = {
  * TODO(alpert): Think of a less-absurd name for createValidatorFunctional.
  *
  */
-
```

Comment:
This also needed to be removed so that the JSDoc comment was actually "attached" to `KhanAnswerTypes`.